src/Uno.Templates/content/unoapp/.template.config/template.json

```diff
@@ -807,7 +807,7 @@
         "cases": [
           {
             "condition": "(tfm == 'net8.0')",
-            "value": "8.0.5"
+            "value": "8.0.7"
```

Comment:
@dansiegel, @jeromelaban can we use latest stable **8.0.14** instead for the MauiVersion?

For example with `CommunityToolkit.Maui` latest stable **8.0.0** if following 
https://platform.uno/docs/articles/external/uno.extensions/doc/Learn/Maui/ThirdParty-MauiCommunityToolkit.html

```
Error: Detected package downgrade: Microsoft.Maui.Controls from 8.0.14 to 8.0.7. Reference the package directly from the project to select a different version. 
error NU1605:  MauiEmbeddingApp.Windows -> MauiEmbeddingApp -> MauiEmbeddingApp.MauiControls -> CommunityToolkit.Maui 8.0.0 -> Microsoft.Maui.Controls (>= 8.0.14) 
error NU1605:  MauiEmbeddingApp.Windows -> Microsoft.Maui.Controls (>= 8.0.7)
```

Reply:
This is a fallback. It should be using the version provided by the Maui workload.